AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/FocusElementLine.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package com.microsoft.accessibilityinsightsforandroidservice;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.view.View;
+import android.view.accessibility.AccessibilityNodeInfo;
+import java.util.HashMap;
+
+public class FocusElementLine {
+  private static final String TAG = "FocusElementLine";
+  private AccessibilityNodeInfo eventSource;
+  private AccessibilityNodeInfo previousEventSource;
+  private int yOffset;
+  private int x1Coordinate;
```

Comment:
It looks like x1Coordiate,y1Coordinate is our start point and that x2Coordinate,y2Coordinate is our end point. That's a bit wordy and is only clear if you look at other code. How would you feel about more descriptive names like xStart, yStart, xEnd, yEnd ?

Reply:
I like using start and end, it's definitely more succinct and still very clear.